### PR TITLE
More specific issue templates to lower the amount of out of scope or invalid issues being made

### PR DESCRIPTION
**Pull request by Kenajcrap**
I find that separating tf2 issues from TC2 bugs is useful because that is a common question asked in issues, also specified that "feature" does not mean balance patch or purely cosmetic changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ labels: 'Type: enhancement'
 assignees: mastercoms
 
 ---
+**NOTE:Make sure the feature being requested does NOT change the gameplay balance in a significant way and is NOT purely a cosmetic choice**
+This poject is not interested in re-balancing the game or changing its looks. Ex. "Buff the Bison", "Bring back the beta muzzle flashes" would not be accepted
```
Comment (micwoj92) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: project

Comment (micwoj92) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: I think it's either "its look" or "it looks" not sure tho

Comment (Kenajcrap) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: "its looks" is valid, but might change that if its sound weird

Comment (micwoj92) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ah, you meant plural.

Comment (Kenajcrap) [reply .github/ISSUE_TEMPLATE/feature_request.md]: yup, but if you looked at it and it was weird, its worth changing.
also fixed the typo

Comment (micwoj92) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I just didn't think of the plural. I thought you meant look as in one big thing.

**Review by treacherousfiend — CHANGES_REQUESTED** [review]
I made a PR without thinking, we could maybe condense the two into just one PR (i'll close mine); maybe we can replace the first note with this:

**NOTE: Anything that changes gameplay (balance changes, etc.) is outside the scope of this project, and your issue will be closed.**

Could also remove the second sentence, or just cull a bit since its summed up pretty well in both of our versions of the first sentence.

**Comment by Kenajcrap** [comment]
how about this? I think its important to keep the examples and the mention to cosmetic changes as those also keep popping up

I don't think "no gameplay changes" is a good way to put it, since QoL SHOULD change gameplay in some way (for the better) but should not change balance
I also think reiterating the position expressed in the readme in the second sentence is necessary since people do not seem to be reading the readme ;-;

**Comment by treacherousfiend** [comment]
Thats fine, i'm still kind of iffy on the existence of the second sentence, but the first sentence is great

**Comment by Kenajcrap** [comment]
i'm doing my best ;-;

**Comment by Yttrium-tYcLief** [comment]
Sometimes shortening a sentence is not the best course, but you're on the right track.

How about:

`Re-balancing and changing the game's looks is beyond the scope of this project. Ex. "Buff the Bison", "Bring back the beta muzzle flashes" would *not* be accepted.`

**Comment by Kenajcrap** [comment]
so no top sentence, just `*NOTE: Re-balancing and changing the game's looks is beyond the scope of this project. Ex. "Buff the Bison", "Bring back the beta muzzle flashes" would not be accepted.*`?
yea. thats fine by me

**Comment by treacherousfiend** [comment]
yeah this seems great! I think that compresses it enough.

**Comment by Kenajcrap** [comment]
closing because #194 made this obsolete

**Pull request closed without merging**